XamarinCommunityToolkit/Behaviors/EffectBehavior.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public class EffectBehavior : BaseBehavior<View>
+    {
+        public static readonly BindableProperty GroupProperty = BindableProperty.Create("Group", typeof(string), typeof(EffectBehavior), null);
```

Comment:
It's slightly better to use **nameof(Group)** instead of a string constant "Group"